js/researches/sentencesLength.js

```diff
@@ -0,0 +1,18 @@
+var wordCount = require( "../stringProcessing/countWords.js" );
+var forEach = require( "lodash/forEach" );
+
+/**
+ * Returns the number of words in a sentence.
```

Comment:
It seems to me that this research doesn't return a number of words, but an array containing the word count per sentence.
Just a small detail :)